tests/acceptance/features/apiSpaces/setQuota.feature

```diff
@@ -0,0 +1,178 @@
+      | User        | Admin       |
+      | User        | Space Admin |
+      | User        | User        |
+      | User        | Guest       |
+      | Guest       | Admin       |
+      | Guest       | Space Admin |
+      | Guest       | User        |
+      | Guest       | Guest       |
+
+
+  Scenario Outline: admin or space admin user sets quota of a space
+    Given the administrator has given "Alice" the role "Space Admin" using the settings api
+    And the administrator has given "Brian" the role "<userRole>" using the settings api
+    And user "Alice" has created a space "Project Jupiter" of type "project" with quota "20"
+    When user "Brian" changes the quota of the "Project Jupiter" space to "100" owned by user "Alice"
+    Then the HTTP status code should be "200"
+    And for user "Alice" the JSON response should contain space called "Project Jupiter" and match
+    """
+    {
+      "type": "object",
+      "required": [
+        "name",
+        "quota"
+      ],
+      "properties": {
+        "name": {
+          "type": "string",
+          "enum": ["Project Jupiter"]
+        },
+        "quota": {
+          "type": "object",
+          "required": [
+            "total"
+          ],
+          "properties": {
+            "total" : {
+              "type": "number",
+              "enum": [100]
+            }
+          }
+        }
+      }
+    }
+    """
+    Examples:
+      | userRole    |
+      | Admin       |
+      | Space Admin |
+
+
+  Scenario Outline: User with role User and Guest can't set the quota of a space
+    Given the administrator has given "Alice" the role "Space Admin" using the settings api
+    And the administrator has given "Brian" the role "<userRole>" using the settings api
+    And user "Alice" has created a space "Project Jupiter" of type "project" with quota "20"
+    And user "Alice" has shared a space "Project Jupiter" with settings:
+      | shareWith | Brian       |
+      | role      | <spaceRole> |
+    When user "Brian" changes the quota of the "Project Jupiter" space to "100"
+    Then the HTTP status code should be "401"
+    And for user "Alice" the JSON response should contain space called "Project Jupiter" and match
+    """
+    {
+      "type": "object",
+      "required": [
+        "name",
+        "quota"
+      ],
+      "properties": {
+        "name": {
+          "type": "string",
+          "enum": ["Project Jupiter"]
+        },
+        "quota": {
+          "type": "object",
+          "required": [
+            "total"
+          ],
+          "properties": {
+            "total" : {
+              "type": "number",
+              "enum": [20]
+            }
+          }
+        }
+      }
+    }
+    """
+    Examples:
+      | userRole | spaceRole |
+      | User     | viewer    |
+      | User     | editor    |
+      | User     | manager   |
+      | Guest    | viewer    |
+      | Guest    | editor    |
+      | Guest    | manager   |
```

Comment:
does space role matter while testing this scenario?

Reply:
Could add a scenario where a user (who is a member of space with <role>) tries to edit quota